share/csm_share/shr/shr_flux_mod.F90

```diff
@@ -267,7 +277,15 @@ real(R8)    :: psix2  ! stability function at ztref reference height
      if (mask(n) /= 0) then
     
         !--- compute some needed quantities ---
-        vmag   = max(umin, sqrt( (ubot(n)-us(n))**2 + (vbot(n)-vs(n))**2) )
+
```

Comment:
Assuming that the flux calculation will stay backwards compatible is not acceptable in my view. We need an if-block to ensure that if ugust is not zero then we do not get unintended consequences for CESM.